code/style.css

```diff
@@ -1,3 +1,62 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: #EEE;
+  font-family: 'Source Code Pro', monospace;
+}
+
+.profile-img {
+  border-radius: 50%;
+  width: 10rem;
+  margin: 1rem;
+}
+
+.user-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.repo-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: stretch;
+  gap: 2rem;
+  padding: 2rem;
+}
+
+.repo-name {
+  background-color: #FFF;
+  padding: 1rem;
+  width: 20rem;
+}
+
+.repo-name a {
+  color: #F4845F;
```

Comment:
Really nice color choice for the links - I think it works really well, pairing this slightly burnt orange with a light gray. Simple yet beautiful palette in my opinion!